src/main/java/org/folio/entlinks/controller/AuthorityController.java

```diff
@@ -40,8 +51,13 @@ public ResponseEntity<AuthorityDto> getAuthority(UUID id) {
   }
 
   @Override
-  public ResponseEntity<AuthorityDtoCollection> retrieveAuthorities(Integer offset, Integer limit, String query) {
-    return ResponseEntity.ok(delegate.retrieveAuthorityCollection(offset, limit, query));
+  public ResponseEntity retrieveAuthorities(Boolean deleted, Boolean idOnly, Integer offset, Integer limit,
+                                            String query, @RequestHeader("Content-type") String contentType) {
+    var collectionDto = Boolean.TRUE.equals(deleted)
+        ? authorityArchiveServiceDelegate.retrieveAuthorityArchives(offset, limit, query, idOnly)
+        : delegate.retrieveAuthorityCollection(offset, limit, query, idOnly);
```

Comment:
Can we add to the services and repositories methods to retrieve only IDs? It could be some projection in repository. It will be more performant operation.